cypress/features/regression/stepSequenceItem.feature

```diff
@@ -28,35 +30,40 @@ Feature: Step Sequence Item component
 
   @positive
   Scenario Outline: I set hiddenCompleteLabel to <hiddenCompleteLabel>
-    When I set hiddenCompleteLabel to "<hiddenCompleteLabel>"
+    When I set hiddenCompleteLabel to <hiddenCompleteLabel> word
       And I select status to "complete"
-    Then hidden label is set to "<hiddenCompleteLabel>"
+    Then hidden label is set to <hiddenCompleteLabel>
     Examples:
-      | hiddenCompleteLabel |
-      | mp150ú¿¡üßä         |
+      | hiddenCompleteLabel     |
+      | mp150ú¿¡üßä             |
 
   @positive
   Scenario Outline: I set hiddenCurrentLabel to <hiddenCurrentLabel>
-    When I set hiddenCurrentLabel to "<hiddenCurrentLabel>"
+    When I set hiddenCurrentLabel to <hiddenCurrentLabel> word
       And I select status to "current"
-    Then hidden label is set to "<hiddenCurrentLabel>"
+    Then hidden label is set to <hiddenCurrentLabel>
     Examples:
-      | hiddenCurrentLabel |
-      | mp150ú¿¡üßä        |
+      | hiddenCurrentLabel      |
```

Comment:
will we check here also special characters ?

Reply:
here it's not require